eng/pipelines/templates/jobs/ci.mgmt.yml

```diff
@@ -15,6 +15,15 @@ jobs:
     steps:
       #- script: "echo $(system.pullrequest.pullrequestnumber), https://github.com/$(build.repository.id), https://github.com/$(build.repository.ID)"
       - template: /eng/pipelines/templates/steps/install-dotnet.yml
+      - task: UseDotNet@2
```

Comment:
I suspect you likely need this in the Test job below as well.

Reply:
Yes, it is needed, wondering why the test run did not fail though.

Reply:
CI failed to trigger because of the bad and() expression.